dotCMS/src/main/java/com/dotcms/rest/api/v1/contenttype/FieldResource.java

```diff
@@ -35,36 +29,78 @@
 import com.dotmarketing.util.UtilMethods;
 import com.liferay.portal.model.User;
 
+import static com.dotcms.util.CollectionsUtils.imap;
+
 @Path("/v1/contenttype/{typeId}/fields")
 public class FieldResource implements Serializable {
 	private final WebResource webResource;
+	private final FieldAPI fieldAPI;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Make "fieldAPI" transient or serializable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1948)

Reply:
why FieldResource is Serializable?